src/components/encounter/ohri-encounter-form.component.tsx

```diff
@@ -88,6 +91,7 @@ export const OHRIEncounterForm: React.FC<OHRIEncounterFormProps> = ({
   const [isFieldInitializationComplete, setIsFieldInitializationComplete] = useState(false);
   const [invalidFields, setInvalidFields] = useState([]);
   const [initValues, setInitValues] = useState({});
+  const [obsGroupCounter, setObsGroupCounter] = useState<Array<RepeatObsGroupCounter | null>>([]);
```

Comment:
The `null` here feels redundant. is there a specific reason why u added it?

Reply:
In the event that there are no obsGroups in the form, this array will always remain empty.

Reply:
I think the initialization u added already takes care of that so there's no need to the null

Reply:
Noted.